src/components/layout/Footer.tsx

```
@@ -0,0 +1,62 @@
+                            </Link>
+                            <Link to="" className='p-[20px] border rounded-[8px]'>
+                                <Linkedin className='h-[24px] w-[24px]' />
+                            </Link>
+                        </div>
+                    </div>
+                </div>
+
+                <hr className="border-[#262626]" />
+
+                <div className='flex justify-between'>
+                    <div className='flex gap-[30px]'>
```

ESLint

[error] Replace `····················<div·className='flex·gap-[30px]'` with `··········<div·className="flex·gap-[30px]"`

(prettier/prettier)